packages/calico/extra/dcos-calico-node.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -xe
+
+# the default configuration path, `/etc/calico/calicoctl.cfg`, of calicoctl is
+# used to simply the interaction with Calico
+/usr/bin/mkdir -p /etc/calico
+/usr/bin/cp /opt/mesosphere/etc/calico/calicoctl.cfg /etc/calico/calicoctl.cfg
+
+# failure to remove the calico-node docker container should not break starting
+# calico node
+/usr/bin/docker rm -f dcos-calico-node || true
+
+/opt/mesosphere/active/calico/bin/dcos-calicoctl apply -f \
```

Comment:
why we changed the name of the binary to `dcos-calicoctl` and not just `calicoctl`?

Reply:
`dcos-calicoctl`  is no longer used and should be removed.